dockstore-webservice/src/main/java/io/dockstore/webservice/core/User.java

```diff
@@ -327,6 +327,10 @@ public void setIsAdmin(boolean isAdmin) {
         this.isAdmin = isAdmin;
     }
 
+    public void setIsCurator(boolean isCurator) {
```

Comment:
There already is a `setCurator`; you shouldn't add another setter.

Reply:
Removed setIsCurator() and refactored to using setCurator()